js/views/search.es6.js

```diff
@@ -0,0 +1,48 @@
+const Parent = window.DDG.base.View;
+
+function Search (ops) {
+
+    this.model = ops.model;
+    this.pageView = ops.pageView;
+    this.template = ops.template;
+
+    Parent.call(this, ops);
+
+    console.log("new search view");
+
+
+    // this._cacheElems() caches jQuery selectors, so the following would be
+    // accessible via: `this.$item` from within this view
+    // and is equivalent to $('.js-search-item')
+
+    this._cacheElems('.js-search', [ 'form', 'input', 'go' ]);
+
+    // this.bindEvents() wires up jQuery selectors to events and their handlers:
+    this.bindEvents([
+      [this.$go, 'click', this._handleSubmit]
+    ]);
+
+    this.bindEvents([
+      [this.$form, 'submit', this._handleSubmit]
+    ]);
+
```

Comment:
Cool, that makes sense. one thing though, don't want to link there from this external repo.